Public/Sdk/Public/Transformers/Transformer.Execute.dsc

```diff
@@ -83,10 +83,28 @@ namespace Transformer {
 
         /** Regex that would be used to extract warnings from the output. */
         warningRegex?: string;
-        
+
         /** Regex that would be used to extract errors from the output. */
         errorRegex?: string;
 
+        /** 
+         * When false (or not set): process output is scanned for error messages line by line;
+         * 'errorRegex' is applied to each line and if ANY match is found the ENTIRE line is reported.
+         *
+         * When true: process output is scanned in chunks of up to 1000 lines; 'errorRegex' is applied to

```

Comment:
Nit: `OutputChunkInLines` is set to 10000

Reply:
thanks, will fix